sdk/core/core/src/context.c

```diff
@@ -0,0 +1,88 @@
+#include <stddef.h>
+
+const long _az_context_max_expiration = 0x7FFFFFFFFFFFFFFF;
```

Comment:
1. Should it be `long` or `int64_t`?
2. (recommendation) `long const` instead of `const long`